src/libraries/Microsoft.PowerFx.Core/Parser/ParseNamedFormulasAndUDFResult.cs

```diff
@@ -0,0 +1,32 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.PowerFx.Core.Errors;
+
+namespace Microsoft.PowerFx.Core.Parser
+{
+    internal class ParseUserDefinitionResult
+    {
+        internal IEnumerable<UDF> UDFs { get; }
+
+        internal IEnumerable<NamedFormula> NamedFormulas { get; }
+
+        internal IEnumerable<TexlError> Errors { get; }
```

Comment:
>internal IEnumerable<TexlError> Errors { get; } [](http://example.com/codeflow?start=8&length=47)

Can we tell which UDF/Named Formula each error is for?

Reply:
No we can't at the moment, we show them based of  spans.